spec/core_functions/color/is_legacy.hrx

```diff
@@ -0,0 +1,237 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(rgb(0 255 0))}
+
+<===> rgb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> rgba/input.scss
+@use "sass:color";
+a {b: color.is-legacy(rgba(72 122 180 / .2))}
+
+<===> rgba/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hwb(0 50% 0%))}
+
+<===> hwb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hsl(110 31% 32%))}
+
+<===> hsl/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hex/input.scss
+@use "sass:color";
+a {b: color.is-legacy(#f2ece4)}
+
+<===> hex/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> named/input.scss
+@use "sass:color";
+a {b: color.is-legacy($color: midnightblue)}
```

Comment:
It looks like this is testing both a named argument _and_ a named color—we should have separate tests for each of these cases (see https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-test-only-one-thing-per-spec)